bundle/Controller/Download.php

```diff
@@ -16,29 +17,37 @@
 use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
+use function in_array;
 use function str_replace;
 
 final class Download extends Controller
 {
+    private array $inlineMimeTypes;
```

Comment:
Needs a fix for PHPStan. (`string[]` PHPDoc)

Reply:
Done